internal/namespaces/instance/v1/testdata/test-server-delete-only-block-volumes.stdout.golden

```diff
@@ -0,0 +1 @@
+✅ Seleted the following resources as well: ip 163.172.146.186, volume 7858de41-9b5f-4e92-8a76-2637b343149b (b_ssd).
```

Comment:
maybe return data formated like a get server

Maybe add a concept of `DeleteServerResponse`, to return not a Success, but a DeleteServerResponse

Reply:
need a golden because UUID change every time